plugins/ida/bap_ida_service.ml

```diff
@@ -79,8 +81,24 @@ let setup_headless_env path =
       if old_path <> "" then Unix.putenv var old_path
     with _ -> ()
 
+let cleanup_minidump () =
+  let is_dump x = String.equal (Filename.extension x) ".dmp" in
```

Comment:
use `Filename.check_suffix n ".dmp"`
in general use the most specialized function available. It conveys you intentions better and is usually more efficient and straightforward